core/templates/pages/topics-and-skills-dashboard-page/modals/unassign-skill-from-topics-modal.component.html

```diff
@@ -4,26 +4,41 @@ <h3>
   </h3>
 </div>
 <div class="modal-body">
-  <div><em>The topics with this skill as their only diagnostic test skill are not eligible for the unassignment.</em></div><br>
-  <span *ngIf="topicsAssignmentsAreFetched && eligibleTopicsForUnassignment.length !== 0">This skill is assigned to these following topics. Tick the topics that you want this skill to be unassigned from.</span>
+  <div *ngIf="topicNamesNotEligibleForUnassignment.length > 0">
+    <div>
+      <p>This skill is used as the only diagnostic test skill in the following topics:</p>
+      <p>{{topicNamesNotEligibleForUnassignment.join(', ')}}</p>
+      <p>
+        Please remove this skill from the diagnostic test of those topics first
+        before unassigning it.
+      </p>
+    </div>
+    <br>
+  </div>
   <div>
-    <span *ngIf="topicsAssignmentsAreFetched && eligibleTopicsForUnassignment.length === 0">No eligible topics for unassignment.</span>
+    <div *ngIf="topicsAssignmentsAreFetched && eligibleTopicNamesForUnassignment.length !== 0">
+      Tick the eligible topics that you want this skill to be unassigned from.
+    </div>
+    <div *ngIf="topicsAssignmentsAreFetched && eligibleTopicNamesForUnassignment.length === 0">
+      This skill is not currently assigned to any topics.
```

Comment:
Sorry, I gave you the wrong correction previously -- "topics" should be "topic".

Reply:
Updated